Source/SwiftLintFramework/Rules/OverrideInExtensionRule.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  OverrideInExtensionRule.swift
+//  SwiftLint
+//
+//  Created by Marcelo Fabri on 10/05/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+public struct OverrideInExtensionRule: ASTRule, ConfigurationProviderRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "override_in_extension",
```

Comment:
This sounds assertive (as in *do this*), instead of restrictive. "disallow_override_in_extensions" maybe?

Reply:
I'm not sure we have a super consistent naming convention. We already have `force_try`, `force_cast`, `fallthrough` and others that aren't restrictive.